src/xo-app/vm/action-bar.js

```diff
@@ -31,7 +31,7 @@ const vmActionBarByState = {
           handler: restartVm,
           pending: includes(vm.current_operations, 'clean_reboot')
         },
-        (isAdmin || !vm.resourceSet) && {
+        {
```

Comment:
If the user isn't an admin and he doesn't have ACLs, they will not have hosts. This case is handled in the vmMigration code.